docs/website/content/docs/v0.1/Getting Started/concepts.md

```diff
@@ -12,17 +12,25 @@ The Metal Controller Manager provides a few custom resources (CRDs) in the manag
 These define a desired deployment environment for Talos.
 This includes things like which kernel to use, kernel args to pass, and the initrd to use.
 
+See the [Environments](/docs/v0.1/Configuration/environments.md) section of our Configuration docs for examples and more detail.
+
 ## Servers
 
 These represent physical machines as resources in the management plane.
 These servers are created when the physical machine PXE boots and completes a "discovery" process in which it registers with the management plane and provides SMBIOS information.
 
+See the [Servers](/docs/v0.1/Configuration/servers.md) section of our Configuration docs for examples and more detail.
+
 ## ServerClasses
 
 ServerClasses are a grouping of the Servers mentioned above.
 These can be used to compose a bank of Servers that are eligible for provisioning.
 
+See the [ServerClasses](/docs/v0.1/Configuration/serverclasses.md) section of our Configuration docs for examples and more detail.
+
 ## Metadata
 
 The metadata server may be familiar to you if you have used cloud environments previously.
 Given Talos machine configurations created by the Sidero Cluster API provider, along with patches specified by editing server resources, metadata is returned to servers who query the metadata server with their UUID as input.

```

Comment:
SHould be "Using Talos machine configurations created by the Sidero Cluster API provider...",  (Using, not Given)

And "metadata is returned to servers who query the metadata server with their UUID as key". (Key not input. Not sure about that...)